src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -612,6 +630,15 @@ else if( Character.isDigit(text.charAt(i)) )
         int letterIndex = text.charAt(i) - '0';
         g.drawImage(getNumbersSmallCaps().getFrame(letterIndex), x, y, menuTextWidth, menuTextHeight, null);
       }
+      else // Assume symbolic
+      {
+        final String charKey = "%./-~,;:!?";
```

Comment:
Parenthesis or at least some sort of bracket would be cool, too

Reply:
Done